.travis.yml

```diff
@@ -9,7 +9,7 @@ before_script:
 - npm install
 script:
 - npm run lint
-- gulp test --production
```

Comment:
This command does two things really, runs the tests but also compiles everything with the `--production` flag meaning that CSS/JS is minified etc. We'd need either add another step after this to compile assets, or add that flag to the `npm test` script.

Reply:
Yep, as discussed, added that in as a separate step.